### ISSUE TITLE
Would it work on Kobra Plus ?

### ISSUE THREAD

**Issue by xromansx** [malebuffy/Kobra-Max-Cobra-Kai-UI-and-Firmware#9]
I would like to try it on Kobra Plus would it work ?

**Comment by xromansx** [comment]
Would be hard to adopt ? LCD is also different ?

**Comment by malebuffy** [comment]
> Would be hard to adopt ? LCD is also different ?

No, actually its not that difficult. LCD should be the same. There is a fork for the Kobra (normal) already. The UI firmware should be similar.

**Comment by Farrit1184** [comment]
I'm fairly certain that the Kobra Plus and the Kobra Max use the same screen firmware. They use the same mainboard firmware.

Edit: Confirmed; I successfully loaded this onto my Kobra Plus without issue.
